spec/routing/survey_routing_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+describe SurveysController, :type => :routing do
+  it "get surveys routes properly" do
+    expect(:get => "/surveys/1").to route_to(
+      :controller => "surveys",
+      :action => "show",
+      :id => "1"
+    )
```

Comment:
read here for the shortcut syntax to do this: https://www.relishapp.com/rspec/rspec-rails/docs/routing-specs/route-to-matcher#passing-route-spec-with-shortcut-syntax